### PR TITLE
[DOCS] Simplifies composite aggregation recommendation

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR makes changes to the Recommendation and Requirements sections of the `Aggregating data for faster performance` page:

* adds a new requirement about influencers
* simplifies two composite aggs-related recommendations and merges them into one.

### CODE REVIEW

File: docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc [hunk 1]
```diff
@@ -83,37 +87,9 @@ type of analysis.
 <<ml-rare-functions,rare>> functions, set the interval to the same value as the 
 bucket span.
 
-* By default, {es} limits the maximum number of terms returned to 10000. For 
-high cardinality fields, the query might not run. It might return errors related 
-to circuit breaking exceptions that indicate that the data is too large. When 
-you use a terms aggregation and the cardinality of a term is high but still 
-significantly less than your total number of documents, use
+* If you have multiple infulencers or partition fields or if your field
```
Comment (droberts195) [inline docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc:90]:
```suggestion
* If you have multiple influencers or partition fields or if your field 
```

File: docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc [hunk 2]
```diff
@@ -83,37 +87,9 @@ type of analysis.
...
-    "service_cardinality": {
-      "cardinality": {
-        "field": "service"
-      }
-    }
-  }
-}
---------------------------------------------------
-// NOTCONSOLE
---
```
Comment (droberts195) [inline docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc:110]: I wonder whether we should leave this example in.  Or even if it's too verbose to leave in the whole example, at least say, "To determine the cardinality of your data use the cardinality aggregation." with "cardinality aggregation" linking to the relevant docs page.

Comment (szabosteve) [reply docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc]: Yes, I think it's better to leave the whole snippet. Added back via [8aa102a](https://github.com/elastic/elasticsearch/pull/89878/commits/8aa102a129a368f17ce303e8d223ca20241245c4).

**Review by droberts195 — APPROVED** [review]
LGTM

**Pull request merged**